docs/integrations/saas-cloud/palo-alto-cortex-xdr.md

```diff
@@ -0,0 +1,191 @@
+  "local_insert_ts": 1677153820311,
+  "matching_service_rule_id": "5",
+  "matching_status": "MATCHED",
+  "name": "IOC (103.36.10.156)",
+  "original_tags": [
+    "DS:PANW/XDR Agent"
+  ],
+  "resolution_status": "STATUS_010_NEW",
+  "severity": "critical",
+  "source": "XDR IOC",
+  "starred": false,
+  "tags": [
+    "DS:PANW/XDR Agent"
+  ]
+}
+```
+
+```json title="Incident Message"
+{
+  "incident_id": "4",
+  "incident_name": null,
+  "creation_time": 1677067079585,
+  "modification_time": 1677154139505,
+  "detection_time": null,
+  "status": "remediated",
+  "severity": "medium",
+  "description": "'IOC (trunapol.xyz)' along with 139 other alerts generated by XDR Agent and XDR IOC detected on host desktop-dlu1nrr involving 2 users",
+  "assigned_user_mail": "ksh@gmail.com",
+  "assigned_user_pretty_name": "ksha",
+  "alert_count": 140,
+  "low_severity_alert_count": 0,
+  "med_severity_alert_count": 25,
+  "high_severity_alert_count": 35,
+  "critical_severity_alert_count": 80,
+  "user_count": 2,
+  "host_count": 1,
+  "notes": null,
+  "resolve_comment": null,
+  "resolved_timestamp": null,
+  "manual_severity": null,
+  "manual_description": null,
+  "xdr_url": "https://sumologic-partner.xdr.us.paloaltonetworks.com/incident-view?caseId=4",
+  "starred": false,
+  "hosts": [
+    "desktop-dlu1nrr:eba0735df9a24a67b95b75ecfdf3c77f"
+  ],
+  "users": [
+    "n/a\\crest_sophos_logs_",
+    "desktop-dlu1nrr\\crest_sophos_logs_"
+  ],
+  "incident_sources": [
+    "XDR Agent",
+    "XDR IOC"
+  ],
+  "rule_based_score": null,
+  "predicted_score": null,
+  "manual_score": null,
+  "aggregated_score": null,
+  "wildfire_hits": 20,
+  "alerts_grouping_status": "Disabled",
+  "mitre_tactics_ids_and_names": [
+    "TA0002 - Execution",
+    "TA0006 - Malfunction Check"
+  ],
+  "mitre_techniques_ids_and_names": [
+    "TA0002 - Execution",
+    "TA0006 - Malfunction Check"
+  ],
+  "alert_categories": [
+    "Domain Name",
+    "File Name",
+    "IP",
+    "Malware"
+  ],
+  "original_tags": [
+    "DS:PANW/XDR Agent"
+  ],
+  "tags": [
+    "DS:PANW/XDR Agent"
+  ]
+}
+```
+
+</details>
+
+## Sample Query
+
+<details><summary>View sample queries</summary>
+
+```sql title="Alerts Triggered"
+_sourceCategory="palo_alto_cortex_xdr" "alert_id" "is_whitelisted"
+| json "alert_id","name","severity","source","host_ip","alert_type","action_pretty","agent_os_type","category","detection_timestamp","is_whitelisted","resolution_status" as alert_id,name,severity,source,host_ip,alert_type,action_pretty,agent_os_type,category,detection_timestamp,is_whitelisted,resolution_status nodrop
+| action_pretty as action
+| where severity matches "{{severity}}"
+| where category matches "{{category}}"
+| where source matches "{{source}}"
+| where action matches "{{action}}"
+| where agent_os_type matches "{{agent_os}}"
+| where alert_type matches "{{alert_type}}"
+| where is_whitelisted matches "{{is_whitelisted}}"
+| count_distinct(alert_id)
+
+```
```

Comment:
Close gap on line 148, then create 1-line gap between the two snippets

Reply:
Thanks Kim, I have updated.